### ISSUE TITLE
Add function stage-directions-incl-speakers

### ISSUE THREAD

**Issue by lehkost** [dracor-org/dracor-api#71]
As proposed by @nilsreiter, let us add another function called ```stage-directions-incl-speakers```. This would be the same as the ```stage-directions``` function, but it would add the speaker strings if they are directly preceding a stage direction. So, for example, this one …

```xml
<sp who="#vroni">
  <speaker>VRONI</speaker>
  <stage>hat den Eimer umgestülpt und sich auf denselben gesetzt.</stage>
  <p>Erzähl weiter von meiner Mutter!</p>
</sp>
```
… would transform into:

```VRONI hat den Eimer umgestülpt und sich auf denselben gesetzt.```

The expected advantage of this added function is that part-of-speech tagging might work better with the subject of the sentence present.

**Comment by cmil** [comment]
Version 0.5.3 adds a resource `/corpora/{corpusname}/play/{playname}/stage-directions-with-speakers`. See, for instance, https://dracor.org/api/corpora/ger/play/alberti-brot/stage-directions-with-speakers.

**Comment by lehkost** [comment]
Is that helpful for POS tagging (and probably other purposes), @nilsreiter?

**Comment by nilsreiter** [comment]
Yes, it should be. I'll play around with it in the coming days. Thanks for the addition!
